.eslintrc.js

```diff
@@ -41,6 +41,8 @@ module.exports = defineConfig({
     "@typescript-eslint",
   ],
   "rules": {
+    "jsdoc/require-returns": "off",
+    "jsdoc/require-jsdoc": "off",
```

Comment:
Keep only the complete disablement ... And I would really prefer that you comment your code accordingly to JSDoc.

Reply:
What do you mean by complete disablement ?

Reply:
`jsdoc/require-jsdoc": "off"` --> disable JSDoc including returns